sys-process/pidof-bsd/pidof-bsd-20050501-r4.ebuild

```diff
@@ -21,7 +22,13 @@ S="${WORKDIR}/pidof"
 PATCHES=( "${FILESDIR}/${P}-gfbsd.patch"
 	"${FILESDIR}/${P}-firstarg.patch"
 	"${FILESDIR}/${P}-pname.patch"
-	"${FILESDIR}/${P}-fbsd10.patch" )
+	"${FILESDIR}/${P}-fbsd11.patch" )
+
+src_prepare() {
+	epatch "${PATCHES[@]}"
+
+	append-ldflags -lkvm
```

Comment:
I dunno about BSD build system but usually append `-l` to LDFLAGS is wrong since LDFLAGS are passed before objects.